engine-comparison/begin-experiment.sh

```diff
@@ -48,16 +35,27 @@ mkdir tmp-configs
 for FENGINE in $FENGINE_CONFIGS; do
   cp $FENGINE tmp-configs/$FENGINE
 done
+
 gcloud compute scp tmp-configs/ ${INSTANCE_NAME}:~/input --recurse --zone=$GCLOUD_ZONE
 rm -r tmp-configs
 
+# These will frequently/usually be defined by the user
+JOBS=${JOBS:-8}
+N_ITERATIONS=${N_ITERATIONS:-5}
+# Send configs
+echo 'BMARKS=$1' > ${SCRIPT_DIR}/dispatcher.config
+echo $'N_ITERATIONS=$N_ITERATIONS\nJOBS=$JOBS' > ${SCRIPT_DIR}/worker.config
```

Comment:
That's definitely a good idea on the readability, and yeah I misused the single quotations here.